vnext/Microsoft.ReactNative/Microsoft.ReactNative.vcxproj

```diff
@@ -13,7 +13,6 @@
     <AppContainerApplication>true</AppContainerApplication>
     <ApplicationType>Windows Store</ApplicationType>
     <ApplicationTypeRevision>10.0</ApplicationTypeRevision>
-    <WindowsTargetPlatformVersion Condition=" '$(WindowsTargetPlatformVersion)' == '' ">10.0.19041.0</WindowsTargetPlatformVersion>
     <WindowsTargetPlatformMinVersion>10.0.17134.0</WindowsTargetPlatformMinVersion>
```

Comment:
Is this min version higher than the others? 🤔

Reply:
Beats me. If we get lucky, it's a mistake and I'll be able to remove that min too after diving into its history.

Reply:
React.Cpp.props should get the SDK values from `Microsoft.ReactNative.WindowsSdk.Default.props`

Reply:
Honestly the whole bit of property sheets could use a refresh. E.g. React.cpp.props means "property sheets included in projects internal to the ABI layer.

Reply:
> React.Cpp.props should get the SDK values from `Microsoft.ReactNative.WindowsSdk.Default.props`

Applying this change.

Reply:
> Is this min version higher than the others? 🤔

@asklar introduced it to allow WinUI3 compatibility. I left it as it is, but added a comment explaining the override.

I think we could set it **conditionally** iif `$(UseWinUI3)` is set to `true`.